tests/libres_tests/res/enkf/test_es_update.py

```diff
@@ -174,6 +184,43 @@ def test_update(setup_case, module, expected_gen_kw):
     assert target_gen_kw == pytest.approx(expected_gen_kw)
 
 
+@pytest.mark.integration_test
+def test_that_posterior_has_lower_variance_than_prior_with_forward_init(copy_case):
+    """This is the poly-case but with FORWARD_INIT set to True, which means
```

Comment:
A small detail, but `FORWARD_INIT TRUE` means that the file should be produced by the forward model, while `FALSE` means that it should be present when the forward model starts. `INIT_FILES` means the sampling is done outside ert.

Reply:
Are not "produced by the foward model" and "the foward-model does the sampling" the same?

Reply:
Maybe my comment was unclear:
`INIT_FILES` -> external sampling of parameters
`FORWARD_INIT FALSE` -> externally sampled parameters are loaded at run path creation
`FORWARD_INIT TRUE` -> externlly sampled parameters are loaded after forward model has completed